backend/controller/controller.go

```diff
@@ -684,6 +690,14 @@ func (s *Service) callWithRequest(ctx context.Context, req *connect.Request[ftlv
 		return nil, err
 	}
 
+	if !verb.IsExported() {
+		for _, caller := range callers {
+			if caller.Module != module {
+				return nil, connect.NewError(connect.CodePermissionDenied, fmt.Errorf("verb %q is not exported", verbRef))
+			}
+		}
+	}
```

Comment:
This currently allows calling from `ftl call ...` and the console, since `callers` will be empty in those cases. In the future, we can lock this down as well with `len(callers) == 0` checks.